### PR TITLE
#95 closest parks filter operating coasters

### PR DESCRIPTION
**Pull request by richardcrete**

### CODE REVIEW

File: translations/messages+intl-icu.fr.yaml [hunk 1]
```diff
@@ -39,8 +39,9 @@ coaster:
         text: 'Score basé sur {count_rating} notes et {count_top} tops'
     infos:
         title: 'Caractéristiques de {name}'
-        closeParksTitle: 'Les meilleurs parcs d''attractions proche de {parkName}'
+        closeParksTitle: 'Les meilleurs parcs d''attractions proches de {parkName}'
         closeParksLink: 'Visualiser {parkName} sur la carte'
+        noCloseParks: 'Aucun parc avec au moins un coaster ayant un score de minimum {score}% à moins de {distance} km de {name}'
```
Comment (florian987) [inline translations/messages+intl-icu.fr.yaml:44]: to be removed

**Review by florian987 — CHANGES_REQUESTED** [review]
should we limit the max number of results to smoething like 5? sometimes list is very long:

image

File: translations/messages+intl-icu.fr.yaml (hunk 1 above)
Comment (richardcrete) [reply translations/messages+intl-icu.fr.yaml]: Done, forgot to remove before commit

**Comment by richardcrete** [comment]
> should we limit the max number of results to smoething like 5? sometimes list is very long:
> 
> image

@florian987 @shARPII how many parks shall we display in this section?

**Comment by florian987** [comment]
> @florian987 @shARPII how many parks shall we display in this section?

let's start with 5 to merge this quickly

**Comment by richardcrete** [comment]
> > @florian987 @shARPII how many parks shall we display in this section?
> 
> let's start with 5 to merge this quickly

Done!

**Review by florian987 — APPROVED** [review]

**Comment by florian987** [comment]
thanks!

**Pull request merged**